ArduPlane/quadplane.cpp

```diff
@@ -1292,3 +1300,37 @@ bool QuadPlane::verify_vtol_land(const AP_Mission::Mission_Command &cmd)
     check_land_complete();
     return false;
 }
+
+void QuadPlane::enable_geofence(void)
```

Comment:
I'd prefer a different name for this such as "check_geofence_autoenable()" otherwise it's implied that this function does something where it actually only does it if configured to do it.